databricks/koalas/tests/test_frame_plot.py

```diff
@@ -15,6 +16,17 @@
 
 
 class DataFramePlotTest(ReusedSQLTestCase, TestUtils):
+
+    @classmethod
```

Comment:
Can we add this same thing into `SeriesPlotTest` too?

Reply:
sure!